contributing-guides/maintainers-guide.md

```diff
@@ -56,35 +56,39 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**,
-  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
-  (4) have been open for at least **24 hours** unless the changes are trivial
+  (2) have all the **review comments addressed**,
+  (3) get **approved reviews by at least two maintainers**, (the second maintainer can merge immediately after approving) and
+  (4) have been open for at least **12 hours** unless the changes are trivial (contains only typo fixes or shell builtin/keyword/quoting syntax fixes)
+  
+  These rules mean that trivial PRs can be merged before 12 hours left.
 
-- If a PR fails to get a review from a second maintainer after a few days,
+- If a PR fails to get a review from a second maintainer after a week,
   the first maintainer should ping others for review. If it still lingers around
   for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
-
-- If the only issues holding up a merge are **trivial fixes**
-  (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
-  **maintainers can make the necessary changes themselves**,
-  and proceed with the merge process.
+  the first maintainer can go ahead and merge it while possibly (if he/she/they) wants dismissing all reviewers who
+  didn't reviewed the PR.
 
 - If a PR **stops getting feedback from the submitter** and is marked as stale
   by [probot-stale](../.github/workflows/stale.yml),
   any maintainer can choose to take over the PR
   and make the necessary changes to get the content ready for merging.
 
 - During the review process, make sure that contributors, especially new ones,
-  are not **overwhelmed with too many change requests**.
+  are not **overwhelmed with too many change requests** (you being a maintainer can
+  write less but more detailed comments in reviews to describe what the problem actually is
+  and how it should be fixed).
   Be mindful of signs of fatigue (less enthusiastic responses, slower reactions),
   and relax review standards if necessary — minor issues can always be fixed later.
+  It doesn't mean that maintainers reviewing PRs from new contributors should violate
+  style guide for simplifying feedback. It means that feedback should be better written
+  with more details.
 
 - When merging PRs, use the **merge strategy that produces a clean Git history**:
   If there's a single commit in the PR,
   or if the multiple commits are not semantically independent changes,
   use the `Squash and merge` method.
-  (Don't forget to clean up the body of the squashed commit message.)
+  Don't forget to clean up the body of the squashed commit message, especially the `Co-authored by:` part.
+  (You can always use the body if you want to include useful information.)
```

Comment:
```suggestion
  (You can always use the body if you want to include useful information.)
  
 Note: For some PRs with reviews and proofreading from contributors or native speakers outside of the tldr -pages organization the `Co-authored by` part must be included in the merge commit message.
```

Reply:
Do u want to tell that for external contributors such `Co-authored` lines should be left in other words?

Reply:
> Do u want to tell that for external contributors such `Co-authored` lines should be left in other words?

Yes, Not for us `Collaborators/members` but for other contributors.